src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/AutoInsert/AutoClosingTagOnAutoInsertProvider.cs

```diff
@@ -39,55 +38,59 @@ internal sealed class AutoClosingTagOnAutoInsertProvider(RazorLSPOptionsMonitor
 
     private static readonly ImmutableHashSet<string> s_voidElementsCaseSensitive = s_voidElements.WithComparer(StringComparer.Ordinal);
 
-    private readonly RazorLSPOptionsMonitor _optionsMonitor = optionsMonitor;
-
     public string TriggerCharacter => ">";
 
-    public bool TryResolveInsertion(Position position, FormattingContext context, [NotNullWhen(true)] out TextEdit? edit, out InsertTextFormat format)
+    public bool TryResolveInsertion(
+        Position position,
+        RazorCodeDocument codeDocument,
+        bool enableAutoClosingTags,
+        [NotNullWhen(true)]
+        out VSInternalDocumentOnAutoInsertResponseItem? autoInsertEdit)
```

Comment:
nit: Please put attributes for parameters on the same line as their parameters. Writing it this way makes it harder to see the signature at a glance.

```suggestion
        [NotNullWhen(true)] out VSInternalDocumentOnAutoInsertResponseItem? autoInsertEdit)
```